app/docker/views/configs/create/createConfigController.js

```diff
@@ -90,4 +89,30 @@ function ($scope, $state, Notifications, ConfigService, Authentication, FormVali
   $scope.editorUpdate = function(cm) {
     $scope.formValues.ConfigContent = cm.getValue();
   };
+
+  function initView() {
+    if ($transition$.params().id === '') {
```

Comment:
I believe that you can simply set `$scope.formValues.displayCodeEditor` to `true` and then return.

It's easier to read than the if/else block.

```javascript
function initView() {
  
  if ($transition$.params().id === '') {
    $scope.formValues.displayCodeEditor = true;
    return;
  }
  
  ConfigService.config($transition$.params().id)
  ...
```

Reply:
Ok, I'm changing the condition also to 
```
    if (!$transition$.params().id) {
```
because of the change to null query parameter.